package.json

```diff
@@ -5,6 +5,7 @@
   "main": "./dist/y-websocket.cjs",
   "module": "./src/y-websocket.js",
   "types": "./dist/src/y-websocket.d.ts",
+  "type": "module",
```

Comment:
Currently, we can't make this change because it would break others using this package.